_maps/icy_caves.json

```diff
@@ -0,0 +1,8 @@
+{
+    "map_name": "Icy Caves",
+    "map_path": "map_files/icy_caves",
+    "map_file": "icy_caves.dmm",
+    "shuttles": {
+    },

```

Comment:
```suggestion
    "environment_traits": [
        {"COLD": true}
    ],
    "armor": "ice",
```

Reply:
I personally think that cold is a shit game mechanic and is just a pain in the ass with very little benefit game play wise.

Setting armor to Ice is good.